### PR TITLE
use Identifier format instead of string format

### PR DESCRIPTION
**Pull request by ChuckHend**
Hopefully addresses https://github.com/tembo-io/pgmq/issues/295

### CODE REVIEW

File: pgmq-extension/sql/pgmq.sql [hunk 1]
```diff
@@ -687,33 +687,33 @@ BEGIN
...
   END IF;
 
   EXECUTE FORMAT(
     $QUERY$
-    SELECT public.create_parent('pgmq.q_%s', '%s', 'native', '%s');
+    SELECT public.create_parent('pgmq.%I', '%s', 'native', '%s');
     $QUERY$,
-    queue_name, partition_col, partition_interval
+    'q_' || queue_name, partition_col, partition_interval
   );
```
Comment (theory) [inline pgmq-extension/sql/pgmq.sql:710]: Arguments to a function should be passed as string literals, not identifiers, right?

```SQL
    SELECT public.create_parent('pgmq.%I', %L, 'native', %L);
```

Honestly, though, you can just call functions:

```sql
    PERFORM public.create_parent(
        'pgmq.' || quote_ident('q_' || queue_name),
        partition_col, partition_interval
    );
````

File: pgmq-extension/sql/pgmq.sql [hunk 2]
```diff
@@ -778,16 +778,16 @@ BEGIN
         retention_keep_table = false,
         retention_keep_index = true,
         automatic_maintenance = 'on'
-    WHERE parent_table = 'pgmq.a_%s';
+    WHERE parent_table = 'pgmq.&I';
```
Comment (theory) [inline pgmq-extension/sql/pgmq.sql:781]:
```suggestion
    WHERE parent_table = 'pgmq.%I';
```

**Review by theory — APPROVED** [review]
Big improvement, left a couple of suggestions.

File: pgmq-extension/sql/pgmq.sql (hunk 1 above)
Comment (ChuckHend) [reply pgmq-extension/sql/pgmq.sql]: good idea!

**Review by theory — APPROVED** [review]
Looks good.

**Pull request merged**